src/Repository/JsonGenericCollectionRepository.php

```diff
@@ -45,6 +41,9 @@ private function init(): void
      */
     public function save(GenericCollection $genericCollection): void
     {
+        if (!file_exists($this->jsonFile)) {
```

Comment:
Maybe you should add `if (!file_exists($this->jsonFile))` into `init()` method and just call it?

Reply:
see other comment, there should be no file created if its turned off

Reply:
it's `true` ;-)